resolwe_runtime_utils.py

```diff
@@ -275,14 +301,233 @@ def checkrc(rc, *args):
     return json.dumps(ret)
 
 
+def export_file(file_path):
+    """Prepend the given parameter with ``export``"""
+
+    if not os.path.isfile(file_path):
+        return error("Referenced file does not exist: '{}'.".format(file_path))
+
+    return "export {}".format(file_path)
+
+
+CHUNK_SIZE = 10000000  # 10 Mbytes
+
+
+class ImportedFormat:
+    """Import destination file format."""
+
+    EXTRACTED = 'extracted'
+    COMPRESSED = 'compressed'
+    BOTH = 'both'
+
+
+def import_file(
+    src,
+    file_name,
+    imported_format=ImportedFormat.BOTH,
+    progress_from=0.0,
```

Comment:
Consider adding here an optional `output_dir` argument, which specifies the destination directory of import. This argument would (1) help prevent overwriting existing files during import. In the case where the source archive contains multiple files or directories, (2) it's easier to see what was imported and (3) `import_file` can then provide a more sensible return value.

Reply:
We have never encountered this use case, so I am not fond of complicating usage. If this is a problem one can always `cd` to a subdir and call `import_file` there. We can add this later if required.